script/gen_crd/gen-kamelets-crd-api-config.json

```diff
@@ -13,8 +13,8 @@
             "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
         },
         {
-          "typeMatchPrefix": "^github\\.com/apache/camel-k/pkg/apis/camel/v1\\.",
-          "docsURLTemplate": "xref:apis/camel-k.adoc#_camel_apache_org_v1_{{.TypeIdentifier}}"
+          "typeMatchPrefix": "^github\\.com/apache/camel-k/v2\\.",
```

Comment:
I don't think this has to be changed, as it related the Kubernetes CRD which is still V1

Reply:
The `v2` should be prefixed before the `pkg`, so it was fixed to contain the `apis/camel/v1`.
Thanks for the review.